users/api/transport.go

```diff
@@ -110,6 +111,20 @@ func MakeHandler(svc users.Service, tracer opentracing.Tracer, logger logger.Log
 		opts...,
 	))
 
+	mux.Post("/users/:id/enable", kithttp.NewServer(
+		kitot.TraceServer(tracer, "enable_user")(enableUserEndpoint(svc)),
+		decodeChangeUserStatus,
+		encodeResponse,
+		opts...,
+	))
+
+	mux.Post("/users/:id/disable", kithttp.NewServer(
+		kitot.TraceServer(tracer, "disable_user")(disableUserEndpoint(svc)),
+		decodeChangeUserStatus,
+		encodeResponse,
+		opts...,
+	))
```

Comment:
Do we need 2 endpoints to change the status? You could fusion them or even use update_user one, right? 

Don't forget to add the API doc here: https://github.com/mainflux/mainflux/blob/master/api/openapi/users.yml

Reply:
We can, but I would say that this is OK at the moment. It's like an endpoint to enable and disable users - without exposing how it's done internally. If we add more status types, we definitely won't add an endpoint for each of them.

Reply:
I have updated the api documentation